src/utils.js

```diff
@@ -52,18 +52,18 @@ export const getConfig = () => {
   const resolvePath = relativePath => path.resolve(path.join(ROOT, relativePath))
 
   // Resolve all paths
+  const distPath = process.env.NODE_ENV === 'development'
```

Comment:
HTML template should use this distPath.

Reply:
Yep! Totally agree. I changed to that. :)